app/src/main/java/org/mozilla/fenix/home/recenttabs/view/RecentTabs.kt

```diff
@@ -299,3 +317,30 @@ private fun RecentTabSubtitle(subtitle: String) {
         maxLines = 1
     )
 }
+
+@Composable
+private fun ThumbnailImage(
+    tabId: String,
+    modifier: Modifier,
+    contentScale: ContentScale,
+    alignment: Alignment
+) {
+    val rememberBitmap = remember(tabId) { mutableStateOf<ImageBitmap?>(null) }
+    val size = LocalDensity.current.run { 116.dp.toPx().toInt() }
+    val request = ImageLoadRequest(tabId, size)
+    val storage = components.core.thumbnailStorage
+    val bitmap = rememberBitmap.value
+    LaunchedEffect(tabId) {
+        rememberBitmap.value = storage.loadThumbnail(request).await()?.asImageBitmap()
+    }
+    if (bitmap != null) {
+        val painter = BitmapPainter(bitmap)
+        Image(
```

Comment:
Maybe add a placeholder image. There's some examples around.

Reply:
I tried looking for an example, but there are none that work for my use case.